benches/README.md

```diff
@@ -0,0 +1,6 @@
+## Benchmarks
+
+Currently, benchmarks following those of [`dashmap`](https://github.com/xacrimon/dashmap/tree/master/benches) and [`hashbrown`](https://github.com/rust-lang/hashbrown/blob/master/benches/bench.rs) are provided. 
+To compare against other hashmap implementations, the benchmarks located in the respective repositories may be executed. 
+Note that `flurry`, like `dashmap`, uses [`criterion`](https://docs.rs/criterion/0.3.1/criterion/) (and [`rayon`](https://docs.rs/rayon/1.3.0/rayon/) for parallel testing), while `hashbrown` uses [`test::bench`](https://doc.rust-lang.org/test/bench/index.html).
+
```

Comment:
It would be good to include instructions here on how to run the benchmarks. And maybe even _also_ how to run the various upstream benchmarks for comparison.

Reply:
I'd maybe add a comment about `criterion` building its own harness. Other than that, you're simply referring to `cargo bench` for the benchmarks inside the project?

For the other benchmarks I'm not familiar enough with the Rust community to know if there is a preferred or particularly clever way to do this - I'd naively set up a "benchmarking project" with all the different maps as dependencies and run the benchmark files? Or check out some respective repository and run `bench` on the project?

Reply:
Ah, sorry, I just meant "what commands do I run?". Like:

    To run the flurry benchmarks, just run

    ```console
    $ cargo bench
    ```
    
    To run the original `dashmap` benchmarks:
    ```console
    $ git clone ...
    $ cd dashmap
    $ cargo bench
    ```
    
    To run the original `hashbrown` benchmarks:
    
    ... etc.